src/App.js

```diff
@@ -14,11 +15,18 @@ class App extends React.Component {
     }
   }
 
+  selectMovie = (id) => {
+    console.log(id);
+    const foundMovie = this.state.movies.find(movie => movie.id === id);
+    
```

Comment:
Smart! I was gonna just take the ID and iterate through the array after all, this is so obviously cleaner!